CHANGES.txt

```diff
@@ -18,6 +18,9 @@ Breaking Changes
 Changes
 =======
 
+ - Added support for hash sums (MD5, SHA1, SHA256). Please
+    see :ref:`sha1 <sha1>`.
```

Comment:
I assumed this would work because I found a reference in the current CHANGES.txt: https://github.com/crate/crate/blob/master/CHANGES.txt#L18 Will remove the reference.